tools/importer-rest-api-specs/components/dataapigeneratorjson/templater_operations.go

```diff
@@ -86,24 +46,8 @@ func codeForOperation(operationName string, operation models.OperationDetails, r
 		resourceId = *operation.ResourceIdName
 	}
 
-	responseObject := ObjectDefinition{}
-	if operation.ResponseObject != nil {
-		responseObject.ReferenceName = operation.ResponseObject.ReferenceName
-		responseObject.Type = ObjectDefinitionType(operation.ResponseObject.Type)
-
-		// TODO
-		//if operation.FieldContainingPaginationDetails == nil {
-		//	code = append(code, fmt.Sprintf(`		public override Type? ResponseObject() => typeof(%[1]s);`, *responseOperationTypeName))
-		//} else {
-		//	code = append(code, fmt.Sprintf(`		public override Type NestedItemType() => typeof(%[1]s);`, *responseOperationTypeName))
-		//}
-	}
-
-	optionsObject := OptionsObject{}
-	options := make([]Option, 0)
+	options := make([]operationModels.Option, 0)
 	if len(operation.Options) > 0 {
-		optionsObject.Name = operationName
-
 		sortedOptionsKeys := make([]string, 0)
```

Comment:
the `options` assignment above will want moving inside of here:

```suggestion
		options :=make([]dataApiModels.Option, 0)
		sortedOptionsKeys := make([]string, 0)
```

since we want to conditionally assign this, as not all operations have Options